theme/sign-in-off/_base.scss

```diff
@@ -1,7 +0,0 @@
-#wb-so {
```

Comment:
Actually, it seems that git insist on keeping it this way unless I make two PRs for this.....